Sources/tart/VMDirectory+Archive.swift

```diff
@@ -0,0 +1,90 @@
+import System
+import AppleArchive
+
+fileprivate let permissions = FilePermissions(rawValue: 0o644)
+
+extension VMDirectory {
```

Comment:
Could you please add a link to the Apple docs? And a note that `lzfse` is a recommended algorithm on Apple Platform.

Reply:
See 46038002fd1bb1aaa4c201558f74d6305bb5ebd6.